src/config.ts

```diff
@@ -59,15 +59,13 @@ const getIssuerEndpoint = async (suffix?: string) => {
     if (issuerEndpoint === defaultEndpoint) {
         issuerEndpoint = await getParameter(`/${stage}/fhirworks-auth-issuer-endpoint`);
     }
-
```

Comment:
Not that I hope this would be a problem, but if we can avoid non-functional changes then it should reduce the possibility of conflicts when pulling down upstream changes.